### ISSUE TITLE
Losses on tensorboard

### ISSUE THREAD

**Issue by rodr-i** [chenhsuanlin/spatial-transformer-GAN#28]
Hello,

could you please share what the loss/validation/graph would look like in tensorboard? Or at least the loos values you print while running the code. I can run the code (indoor), but the losses are basically constant, it seems the network is stuck

Thanks

**Comment by MeNinaa** [comment]
Not sure if my losses are correct, but they look like this
![image](https://user-images.githubusercontent.com/7231598/62533739-57fa9080-b83f-11e9-9e8d-11b7561b7a2c.png)

I hope this helps

**Comment by chenhsuanlin** [comment]
The loss shouldn't be constant. During the pretraining stage, the loss curve would resemble that from @mariaito, and during adversarial training it should be oscillating all the way. You can also check that the gradient norm (logged into the terminal) should converge to 1 pretty quickly.
